databases/de-database-schema/src/main/constraints/46_genome_ref.sql

```diff
@@ -0,0 +1,15 @@
+SET search_path = public, pg_catalog;
+
+--
+-- Uniqueness constraint on reference genome name.
+--
+ALTER TABLE ONLY genome_reference
+ADD CONSTRAINT genome_ref_mame_unique
```

Comment:
It's just in the name of a constraint so it doesn't change functionality, but this says `mame`, not `name`